README.md

```diff
@@ -19,6 +19,8 @@ $ yarn dev
 NEXT_PUBLIC_FIREBASE_API_KEY=
 NEXT_PUBLIC_FIREBASE_AUTH_DOMAIN=
 NEXT_PUBLIC_FIREBASE_PROJECT_ID=
+NEXT_PUBLIC_LOGFLARE_KEY=
```

Comment:
How public are these two really are? 🤔 I briefly Googled but couldn't find any explicit mention.

Reply:
@chasers I'd love your feedback here. How should I be handling this? I initially made these available on the client-side, but is that the best practice?

Reply:
For Logflare, the api_key is an ingest api_key which can easily be changed. And the source is the source token (or source ID) which can also easily be changed by adding a new source. Theoretically, someone could spam your logs by making these available on the client but that's the case for Google Analytics, Mixpanel, et al. Neither key allows for any sort of Logflare account management or querying.

Reply:
Thank you for the info!